cmd/hdfs/df.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"text/tabwriter"
+
+	"github.com/colinmarc/hdfs"
+)
+
+func df(humanReadable bool) {
+	client, err := getClient("")
+	if err != nil {
+		fatal(err)
+	}
+
+	var fs hdfs.FsInfo
+
+	fs, err = client.StatFs()
+	if err != nil {
+		fatal(err)
+	}
+
+	tw := dfTabWriter()
+	fmt.Fprintf(tw, "Filesystem \tSize \tUsed \tAvailable \t Use\n")
+	if humanReadable {
+		fmt.Fprintf(tw, "%v \t%v \t%v \t%v \t%d%%\n",
+			os.Getenv("HADOOP_NAMENODE"),
+			formatBytes(fs.Capacity()),
+			formatBytes(fs.Used()),
+			formatBytes(fs.Remaining()),
+			100 * fs.Used() / fs.Capacity())
+	} else {
+		fmt.Fprintf(tw, "%v \t%v \t %v \t %v \t%d%%\n",
+			os.Getenv("HADOOP_NAMENODE"),
+			fs.Capacity(),
+			fs.Used(),
+			fs.Remaining(),
+			100 * fs.Used() / fs.Capacity())
+	}
+	tw.Flush()
+}
+
+func dfTabWriter() *tabwriter.Writer {
```

Comment:
I don't see a reason to have this split out as a separate function - just put it inline.